test/integration/helpers.js

```diff
@@ -487,9 +488,13 @@ async function runMochaWatchJSONAsync(args, opts, change) {
  */
 function touchFile(filepath) {
   fs.ensureDirSync(path.dirname(filepath));
-  touch.sync(filepath);
+  try {
+    fs.utimesSync(filepath, touchRef, touchRef);
+  } catch (e) {
+    const fd = fs.openSync(filepath, 'a');
+    fs.closeSync(fd);
+  }
 }
-
```

Comment:
[Style] Was removing this blank line intentional? (not at all a blocker for me, though I do prefer it there a bit 😄)